<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="langComboBox">
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="themeComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="SaveAs1"/>
    <addaction name="OpenAs1"/>
    <addaction name="Exit1"/>
   </widget>
   <widget class="QMenu" name="EditMenu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="Copy"/>
    <addaction name="Paste"/>
    <addaction name="Cut"/>
    <addaction name="Clear"/>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
    <addaction name="Search"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="EditMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="Copy">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="Paste">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="Cut">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="Clear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="Undo">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="Redo">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="Search">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="actionTXT">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionJSON">
   <property name="text">
    <string>JSON</string>
   </property>
  </action>
  <action name="actionTXT_2">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionJSON_2">
   <property name="text">
    <string>JSON</string>
   </property>
  </action>
  <action name="SaveAs1">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="OpenAs1">
   <property name="text">
    <string>Открыть как...</string>
   </property>
  </action>
  <action name="Exit1">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Undo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Clear</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Redo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
